grid.v3/msg.go

```diff
@@ -43,6 +44,35 @@ type ActorStart struct {
 	Data []byte
 }
 
+type ackmsg struct {
```

Comment:
this msg is both the message an it's codec.  It also doesn't unmashal anything as it can just return the global Ack message.